lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php

```diff
@@ -62,18 +64,25 @@ public static function parse(ParserState $oParserState) {
 	}
 
 
-	public function setSelectors($mSelector) {
+	public function setSelectors($mSelector, $oList = NULL) {
 		if (is_array($mSelector)) {
 			$this->aSelectors = $mSelector;
 		} else {
 			$this->aSelectors = explode(',', $mSelector);
 		}
 		foreach ($this->aSelectors as $iKey => $mSelector) {
 			if (!($mSelector instanceof Selector)) {
-				if (!Selector::isValid($mSelector)) {
-					throw new UnexpectedTokenException("Selector did not match '" . Selector::SELECTOR_VALIDATION_RX . "'.", $mSelector, "custom");
+				if ($oList === NULL || !($oList instanceof KeyFrame)) {
+					if (!Selector::isValid($mSelector)) {
+						throw new UnexpectedTokenException("Selector did not match '" . Selector::SELECTOR_VALIDATION_RX . "'.", $mSelector, "custom");
+					}
+					$this->aSelectors[$iKey] = new Selector($mSelector);
+				} else {
+					if (!KeyframeSelector::isValid($mSelector)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Sabberworm\CSS\Property\KeyframeSelector' in method 'setSelectors'.](https://app.codacy.com/app/sabberworm/PHP-CSS-Parser/pullRequest?prid=4484969)